jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/SqlGenerationVisitor.java

```diff
@@ -49,14 +49,14 @@ protected void checkDialectAliases() {
         // Check if dialect provided invalid aliases, which would never be applied.
         for (final ScalarFunction function : this.dialect.getScalarFunctionAliases().keySet()) {
             if (!function.isSimple()) {
-                throw new RuntimeException("The dialect " + SqlDialect.getPublicName()
+                throw new RuntimeException("The dialect " + this.dialect.getName()
```

Comment:
Let's replace them or it's going to be a "new" code smell. IllegalArgumentException?

Reply:
Changed to `UnsupportedOperationException`-